src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Extensions/ParenthesizedExpressionSyntaxExtensions.cs

```diff
@@ -729,6 +729,10 @@ public static bool CanRemoveParentheses(this ParenthesizedPatternSyntax node)
             if (node.Parent is IsPatternExpressionSyntax)
                 return true;
 
+            // x is [ ..(<...>) ]  ->  x is [ ..<...> ]
+            if (node.Parent is SlicePatternSyntax)
+                return true;
```

Comment:
is there never any precedence concern?

Reply:
Correct. `..<pattern>` and `..(<pattern>)` should always mean the same thing.